packages/scratch-gui/src/lib/cloud-manager-hoc.jsx

```diff
@@ -111,12 +111,16 @@ const cloudManagerHOC = function (WrappedComponent) {
                 canModifyCloudData,
                 cloudHost,
                 projectId,
-                username,
                 hasCloudPermission,
                 isShowingWithId,
                 onShowCloudInfo,
                 /* eslint-enable no-unused-vars */
+
                 vm,
+
+                // Intentionally propagating this one since it's used in the
```

Comment:
...since it's used in the... _aaaaarrgh_

> Perhaps he was dictating?

Reply:
whoops 😄

Reply:
Took me a while but I eventually saw what you

in your other comment